README.md

```diff
@@ -466,54 +466,54 @@ client.delete_secret(API_KEY, 'my-secret-id')
 response = client.application_v2.create_application({name='Example App', type='voice'})
 ```
 
-Docs: [https://developer.nexmo.com/api/application.v2#createApplication](https://developer.nexmo.com/api/application.v2#createApplication?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library#create-an-application)
+Docs: [https://developer.vonage.com/api/application.v2#createApplication](https://developer.vonage.com/api/application.v2#createApplication?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library#create-an-application)
 
 ### Retrieve a list of applications
 
 ```python
 response = client.application_v2.list_applications()
 ```
 
-Docs: [https://developer.nexmo.com/api/application.v2#listApplication](https://developer.nexmo.com/api/application.v2#listApplication?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library#retrieve-your-applications)
+Docs: [https://developer.vonage.com/api/application.v2#listApplication](https://developer.vonage.com/api/application.v2#listApplication?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library#retrieve-your-applications)
```

Comment:
This URL should be updated. It only works because Github will redirect it, but it should point to the correct URL